src/SeleniumLibrary/keywords/alert.py

```diff
@@ -52,7 +53,7 @@ def alert_should_be_present(
         self,
         text: str = "",
         action: str = ACCEPT,
-        timeout: Union[float, str, None] = None,
+        timeout: Optional[timedelta] = None,
```

Comment:
`Optional[]` is unnecessary when there's `None` as a default value. Even MyPy doesn't need it unless you run it with `--no-implicit-optional`:
https://mypy.readthedocs.io/en/stable/command_line.html?highlight=optional#cmdoption-mypy-no-implicit-optional

Reply:
I think Optional is best practice if None is used. Types should not contain None as part of the type.

Reply:
And the default is also changing in mypy https://github.com/python/mypy/issues/9091